README.md

```diff
@@ -158,9 +158,12 @@ Note: We only support Docker on Linux and Windows. On Windows, Docker needs to b
 #### Build Wheels
 
 1. Open a shell in the PyPRT git root
+1. Create the base image for the desired build toolchain (adapt to your desired Python version):
+   * Linux: `docker build --rm -f envs/centos7/base/Dockerfile -t pyprt-base:centos7-py3.6 --build-arg PY_VER=3.6 --build-arg USER_ID=$(id -u) --build-arg GROUP_ID=$(id -g) .`
+   * Windows: `docker build --rm -f envs\windows\base\Dockerfile -t pyprt-base:windows-py3.6 --build-arg PY_VER=3.6 .`
```

Comment:
For wheels on Windows, we use a different base image dockerfile: `Dockerfile-py` (prebuilt windows images provided by the python community)